centos-ci/libtask.sh

```diff
@@ -18,9 +18,6 @@ prepare_job() {
 
     . ~/rsync-password.sh
 
-    # Ensure we're operating on a clean base
-    (cd ${buildscriptsdir} && git clean -dfx && git reset --hard HEAD)
```

Comment:
Where else are we resetting?  `git grep git.*reset` only has this as a hit.  Are you thinking it's done by Jenkins?  I don't think so...from [the JJB docs](http://docs.openstack.org/infra/jenkins-job-builder/scm.html)

```
clean (bool) –

Clean after checkout (default false)
```

which we're not using.